In [5]:
# Testing iFF708
!memote report snapshot /Users/Admin/github-classroom/27410/group-assingment-team-4/yeast-GEM-8.7.0/model/yeast-GEM2.xml

In [1]:
!memote report snapshot /Users/Admin/github-classroom/27410/group-assingment-team-4/yeast-GEM-8.7.0/model/yeast-GEM2.xml --filename /Users/Admin/github-classroom/27410/group-assingment-team-4/yeast-GEM-8.7.0/model/yeast-GEM2.html

============================= test session starts =============================
platform win32 -- Python 3.7.16, pytest-4.6.11, py-1.11.0, pluggy-0.13.1
rootdir: C:\Users\Admin\github-classroom\27410\group-assingment-team-4\analysis
plugins: anyio-3.7.1
collected 146 items / 1 skipped / 145 selected

test_annotation.py FF.FFFFFFFFFFFFFFFFFFFFFFFFFFFFFF...FFFFF...F.F..F.F. [ 36%]
..F.F..FFF..                                                             [ 44%]
test_basic.py ....F.F........F.F.F.FF                                    [ 60%]
test_biomass.py .F.....F.F                                               [ 67%]
test_consistency.py FFF.........sssss..FFFFFF.FFF                        [ 86%]
test_essentiality.py s                                                   [ 87%]
test_growth.py s                                                         [ 88%]
test_matrix.py ....                                                      [ 91%]
test_sbml.py ..                                           

The current solver interface glpk doesn't support setting the optimality tolerance.
Writing snapshot report to '/Users/Admin/github-classroom/27410/group-assingment-team-4/yeast-GEM-8.7.0/model/yeast-GEM2.html'.


In [2]:
!memote report snapshot /Users/Admin/github-classroom/27410/group-assingment-team-4/yeast-GEM-8.7.0/model/iFF708.xml --filename /Users/Admin/github-classroom/27410/group-assingment-team-4/yeast-GEM-8.7.0/model/iFF708.html

============================= test session starts =============================
platform win32 -- Python 3.7.16, pytest-4.6.11, py-1.11.0, pluggy-0.13.1
rootdir: C:\Users\Admin\github-classroom\27410\group-assingment-team-4\analysis
plugins: anyio-3.7.1
collected 155 items / 1 skipped / 154 selected

test_annotation.py FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF [ 34%]
FFFFFFFFFF..                                                             [ 41%]
test_basic.py ....F.FF.....FFFFF...FF                                    [ 56%]
test_biomass.py .FF........FF....FF                                      [ 69%]
test_consistency.py FFFssssssssssssssss.FFFFF.FFF                        [ 87%]
test_essentiality.py s                                                   [ 88%]
test_growth.py s                                                         [ 89%]
test_matrix.py ....                                                      [ 91%]
test_sbml.py ..                                           

The current solver interface glpk doesn't support setting the optimality tolerance.
Writing snapshot report to '/Users/Admin/github-classroom/27410/group-assingment-team-4/yeast-GEM-8.7.0/model/iFF708.html'.


In [3]:
!memote report snapshot /Users/Admin/github-classroom/27410/group-assingment-team-4/yeast-GEM-8.7.0/model/iND750.xml --filename /Users/Admin/github-classroom/27410/group-assingment-team-4/yeast-GEM-8.7.0/model/iND750.html

============================= test session starts =============================
platform win32 -- Python 3.7.16, pytest-4.6.11, py-1.11.0, pluggy-0.13.1
rootdir: C:\Users\Admin\github-classroom\27410\group-assingment-team-4\analysis
plugins: anyio-3.7.1
collected 146 items / 1 skipped / 145 selected

test_annotation.py ...FFFFFFFF..FFFF..FFFFFFFFFFFFFFF..FF..F........FFF. [ 36%]
FFFFF.FFFF..                                                             [ 44%]
test_basic.py ......FF.......F...F.FF                                    [ 60%]
test_biomass.py .F....FF.F                                               [ 67%]
test_consistency.py ...FFFFF..sssssssFF.FFFFF.FFF                        [ 86%]
test_essentiality.py s                                                   [ 87%]
test_growth.py s                                                         [ 88%]
test_matrix.py ....                                                      [ 91%]
test_sbml.py ..                                           

The current solver interface glpk doesn't support setting the optimality tolerance.
Writing snapshot report to '/Users/Admin/github-classroom/27410/group-assingment-team-4/yeast-GEM-8.7.0/model/iND750.html'.


In [4]:
!memote report snapshot /Users/Admin/github-classroom/27410/group-assingment-team-4/yeast-GEM-8.7.0/model/iMM904.xml --filename /Users/Admin/github-classroom/27410/group-assingment-team-4/yeast-GEM-8.7.0/model/iMM904.html

============================= test session starts =============================
platform win32 -- Python 3.7.16, pytest-4.6.11, py-1.11.0, pluggy-0.13.1
rootdir: C:\Users\Admin\github-classroom\27410\group-assingment-team-4\analysis
plugins: anyio-3.7.1
collected 146 items / 1 skipped / 145 selected

test_annotation.py ...FFFFFFFF..FFFFF.FFFFFFFFFFFFFFF..FF..F...F....FFF. [ 36%]
FFFFF.FFFF..                                                             [ 44%]
test_basic.py ....F.FF.......F...F.FF                                    [ 60%]
test_biomass.py .F....FF.F                                               [ 67%]
test_consistency.py ...FFFFF..s.sssssFFFFFFFF.FFF                        [ 86%]
test_essentiality.py s                                                   [ 87%]
test_growth.py s                                                         [ 88%]
test_matrix.py ....                                                      [ 91%]
test_sbml.py ..                                           

The current solver interface glpk doesn't support setting the optimality tolerance.
Writing snapshot report to '/Users/Admin/github-classroom/27410/group-assingment-team-4/yeast-GEM-8.7.0/model/iMM904.html'.


Memote results

| Parameter    | yeast8.7.0 | iMM904 | iFF708 | iND750 |
| -------- | ------- | ------- | ------- | ------- |
| Total metabolites | 2807     | 1226     | 796     | 1059     |
| Total reactions | 4133     | 1577     | 1379     | 1266     |
| Total genes | 1163     | 905     | 619     | 750     |
| Total purely metabolic reactions | 2404     | 1003     | 1167     | 841     |
| Enzyme complexes | 208     | 136     | 21     | 86     |
| Mass balance | 93,8%     | 96,0%     | 0,0%     | 97,3%     |
| Charge balance | 98,2%     | 98,5%     | 100%     | 100%     |
| Metabolite connectivity | 100%     | 100%     | 100%     | 100%     |
| Presence of metabolite/gene/reactions annotation | 100%     | 100%     | 0%     | 100%     |
| Metabolite general SBO presence | 100%     | 100%     | 0%     | 100%     |
| Total memote score | 69%     | 85%     | 19%     | 86%     |